skimage/metrics/simple_metrics.py

```diff
@@ -158,3 +161,99 @@ def peak_signal_noise_ratio(image_true, image_test, *, data_range=None):
 
     err = mean_squared_error(image_true, image_test)
     return 10 * np.log10((data_range ** 2) / err)
+
+
+def _pad_to(arr, shape):
+    """Pad an array with trailing zeros to a given target shape.
+
+    Parameters
+    ----------
+    arr : ndarray
+        The input array.
+    shape : tuple
+        The target shape.
+
+    Returns
+    -------
+    padded : ndarray
+        The padded array.
+
+    Examples
+    --------
+    >>> _pad_to(np.ones((1, 1), dtype=int), (1, 3))
+    array([[1, 0, 0]])
+    """
+    if not all(s >= i for s, i in zip(shape, arr.shape)):
+        raise ValueError(f"Target shape {shape} cannot be smaller than input"
+                         f"shape {arr.shape} along any axis.")
+    padding = [(0, s-i) for s, i in zip(shape, arr.shape)]
+    return np.pad(arr, pad_width=padding, mode='constant', constant_values=0)
+
+
+def normalized_mutual_information(im_true, im_test, *, bins=100):
```

Comment:
For consistency with `normalized_root_mse` and `peak_signal_noise_ratio`, may be `im_true` and `im_test` musr be renamed to `image_true` and `image_test`.

BTW, I see that `mean_squared_error` uses `image0` and `image1` :confused:, this can be a new PR :wink:.

Reply:
I think we decided that symmetric measures would use image0 and image1, and asymmetric ones would use image_true and image_test. So I think that argues that these should be image0 and image1?

Reply:
CC @scikit-image/core

Reply:
Interesting! Perhaps we can simply adopt the true/test for both, even though it is slightly odd for the prior case. I have no strong intuition here.

Reply:
LOL I hunted down the [original discussion](https://github.com/scikit-image/scikit-image/pull/4187#issuecomment-537842032), which happened just over a year ago, and contains this nugget:

> After discussing with Stéfan in the community meeting this week, we settled on the following:
>
> - for comparison functions, `image0` and `image1` where the metric is in a strong sense symmetric, and `image_true` and `image_test` where it is not. (though this could be `true_image` and `test_image`, come to think of it)

😂

Reply:
Interesting, indeed! I'm in favour of sticking with what you folks settled a year ago, so `image0` and `image1` in this case.

Reply:
Given that these are not named keywords, I think it's fine to use the argument name to indicate the nature of what's happening.  But, again, I don't feel strongly either way---as long as the documentation is clear, this shouldn't be too confusing.